src/base/base.pri

```diff
@@ -53,7 +53,8 @@ HEADERS += \
     $$PWD/unicodestrings.h \
     $$PWD/torrentfilter.h \
     $$PWD/scanfoldersmodel.h \
-    $$PWD/searchengine.h
+    $$PWD/searchengine.h \
+    $$PWD/rss/utorrentrssdata.h
```

Comment:
It's **qmake** project file! We manage this in different ways, and often manually.